tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -124,6 +102,24 @@ describe(__filename, () => {
       expect(root.find(AddonReviewCard).at(0)).toHaveProp('review', null);
     });
 
+    it('renders an AddonSummaryCard with an addon', () => {
+      const addon = fakeAddon;
+      dispatchAddon(addon);
+      const root = render();
+
+      const summary = root.find(AddonSummaryCard);
+      expect(summary).toHaveProp('addon', createInternalAddon(addon));
+      expect(summary).toHaveProp('headerText', `Reviews for ${addon.name}`);
+    });
+
+    it('renders an AddonSummaryCard without an addon', () => {
+      const root = render({ addon: null });
```

Comment:
I think you meant this, right?

```suggestion
      const root = render();
```